CommMicro.py

```diff
@@ -65,86 +65,113 @@ def getPath(fileName):
         self.xfDisp = Display(ui_filename=getPath("MicPlot.ui"))
 
         # Show message on gui
-        self.ui.AcqProg.setText("Select 1 CM and 1 cavity at a time for commissioning. \nLimit plotted waveforms to 30 sec.")         
+        self.ui.AcqProg.setText(
+            "Select 1 CM and 1 cavity at a time for commissioning. \nLimit plotted waveforms to 30 sec.")

```

Comment:
Is this limit enforced somewhere?

Reply:
I think 1 CM is limited by using spin box and 1 cavity is no longer true - you can select up to 4 cavities with the check boxes.

Reply:
I'm not sure about that 30sec limit for the plots. I don't see that limitation in the code anywhere. Maybe to keep things from slowing down?

Reply:
I'll get rid of the text label